src/type-utils.ts

```diff
@@ -61,7 +61,7 @@ export const validate = {
 	text: (length?: number) =>
 		checkRequired((value: any) => {
 			if (typeof value !== 'string') {
-				throw new Error('is not a string: ' + value);
+				throw new Error('is not a string: ' + JSON.stringify(value));
```

Comment:
I'd prefer
```suggestion
				throw new Error('is not a string: ' + typeof value);
```
rather than returning the full thing back (which may or may not be safe to do)

Reply:
sounds good